docker-compose.yml

```diff
@@ -15,11 +13,8 @@ services:
 
   kafka:
     image: confluentinc/cp-enterprise-kafka:5.0.0
-    hostname: kafka
-    container_name: kafka
     ports:
       - '9092:9092'
-      - '29092:29092'
```

Comment:
29092 is the port that other containers use to connect to this Kafka broker

Reply:
you don't need to expose the port that's used within the docker network, only for external connections from the host, and that's covered by the listener on 9092

Reply:
True!  It's ok to remove 29092